<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit</class>
 <widget class="QMainWindow" name="Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактировать</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cancelbut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="okbut">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>170</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_line">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="carb_line">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>90</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="prot_line">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>40</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lip_line">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>65</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ccal_line">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>115</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="glyc_line">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>140</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="deletebut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="prot_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Белки</string>
    </property>
   </widget>
   <widget class="QLabel" name="lip_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Жиры</string>
    </property>
   </widget>
   <widget class="QLabel" name="carb_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Углеводы</string>
    </property>
   </widget>
   <widget class="QLabel" name="ccal_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>115</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Калорийность, ккал</string>
    </property>
   </widget>
   <widget class="QLabel" name="glyc_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Гликемический индекс</string>
    </property>
   </widget>
   <widget class="QLabel" name="subs_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Продукт</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>search_line</tabstop>
  <tabstop>prot_line</tabstop>
  <tabstop>lip_line</tabstop>
  <tabstop>carb_line</tabstop>
  <tabstop>ccal_line</tabstop>
  <tabstop>glyc_line</tabstop>
  <tabstop>deletebut</tabstop>
  <tabstop>okbut</tabstop>
  <tabstop>cancelbut</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelbut</sender>
   <signal>clicked()</signal>
   <receiver>Edit</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
